xds/src/main/java/io/grpc/xds/client/XdsClientImpl.java

```diff
@@ -109,7 +112,9 @@ public XdsClientImpl(
       Supplier<Stopwatch> stopwatchSupplier,
       TimeProvider timeProvider,
       MessagePrettyPrinter messagePrinter,
-      Object securityConfig) {
+      Object securityConfig,
+      String target,
```

Comment:
Don't pass `target` into XdsClientImpl. It isn't using it; it is just echoing it right back out. Instead, `target` should be remembered by `XdsClientMetricReporterImpl` and "injected" while reporting.

Reply:
Done